frontends/pytorch/csrc/builder/node_importer.cpp

```diff
@@ -123,6 +123,14 @@ void NodeImporter::importPrimNode(Node *node, MlirBlock appendToBlock) {
     return;
   }
 
+  if (kind == c10::prim::ListConstruct) {
+    MlirOperation operation = createMlirOperationAtEnd(
+        appendToBlock, "basicpy.build_list", loc, npcompListTypeGet(context),
```

Comment:
this can be tested analogously to frontends/pytorch/test/graph_import/tuple.py (create graph_import/list.py).

the cheat sheet here is
- `ivalue_importer.cpp` is tested by `frontends/pytorch/test/module_import/`
- `node_importer.cpp` is tested by `frontends/pytorch/test/graph_import/`

I'll rename the directories after you land this PR.